components/_common_/Button/Button.js

```diff
@@ -41,8 +41,9 @@ class Button extends Component {
       // eslint-disable-next-line no-console
       console.log('Analytics Disabled', props.analyticsObject);
     }
-
-    return props.onClick;
+    if (typeof props.onClick === 'function') {
```

Comment:
Instead of this, let's set the default value of the onClick prop to be `() => {}`

Reply:
Honestly I did not notice the default prop, Does creating an anon function better then a type check?
just curious I know for the scope of this project it doesn't matter.

Reply:
It's just less terse, imo. We can guarantee that it's always a function.